helix-view/Cargo.toml

```diff
@@ -20,6 +20,7 @@ helix-core = { version = "0.6", path = "../helix-core" }
 helix-lsp = { version = "0.6", path = "../helix-lsp" }
 helix-dap = { version = "0.6", path = "../helix-dap" }
 crossterm = { version = "0.25", optional = true }
+helix-vcs = { version = "0.6", path = "../helix-vcs" }
```

Comment:
```suggestion
helix-vcs = { version = "0.6", path = "../helix-vcs" }
crossterm = { version = "0.25", optional = true }
```

Reply:
It there some kind of formatter (taplo?) that is conventionally used by helix or some kind of formatting styles so I can make sure that such reviews are not necessary in the future?

Reply:
I think generally the `helix-X` crates are grouped together.

Reply:
ok thats pretty standard I will try to keep this in mind in the future